sys/net/gnrc/network_layer/ipv6/nib/_nib-6ln.h

```diff
@@ -35,8 +35,7 @@ extern "C" {
 
 static inline uint32_t _now_min(void)
 {
-    return (uint32_t)((xtimer_now_usec64() / (US_PER_SEC * SEC_PER_MIN)) &
-                      UINT32_MAX);
+    return evtimer_now_msec() / (MS_PER_SEC * SEC_PER_MIN);
```

Comment:
Even with the removal of the explicit truncation in `evtimer_now_msec()` the value space stays reduced. Might I suggest, to just define a function `evtimer_now_min()`?

Reply:
Good idea!